interface/frontend/src/components/open-image/ImageSeries.vue

```diff
@@ -141,6 +172,9 @@
 </script>
 
 <style lang="scss" scoped>
+  .case-btn {
+    margin-left: 1% 
```

Comment:
Bootstrap has `ml-1`, `ml-2` etc. for margins. Check [this](https://v4-alpha.getbootstrap.com/utilities/spacing/).